tests/test_serializers.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+"""
+Tests for the `super-csv` mixins module.
+"""
+
+import json
+
+from django.test import TestCase
+
+from super_csv.serializers import CSVOperation, CSVOperationSerializer
+
+
+class SerializerTestCase(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.data = {
+            "total_rows": 3,
+            "processed_rows": 4,
+            "saved_rows": 5
+        }
+
+        self.operation = CSVOperation.record_operation('some_name', 'course_id', 'save', json.dumps(self.data))
+
+    def tearDown(self):
+        super().tearDown()
+        CSVOperation.expire_data(-1)
```

Comment:
You shouldn't need to do this. iirc TestCase automatically removes any django model instances created during testing

Reply:
This one store into the file storage and does not get delete. Only model get roll back.